packer/manifest.yml

```diff
@@ -0,0 +1,17 @@
+author: Itamar
+email: itamar@redislabs.com
+description: enhance redis with json capabilites
+homepage: rejson.io
+license: BSD
+command_line_args: ""
+min_redis_version: 4.0
+min_rlec_version: 5.2
+capabilities:
+    - cap_types
```

Comment:
remove the prefixes. this is a freeform list of strings that is not RLEC specific.

Reply:
@dvirsky / @swilly22 - Note that some capabilities _are_ RLEC specific